packages/wrangler/src/hyperdrive/client.ts

```diff
@@ -9,15 +9,14 @@ export type HyperdriveConfig = {
 	caching: CachingOptions;
 };
 
-export type Origin = {
+export type PublicOrigin = {
 	host?: string;
 	port?: number;
-};
-
-export type PublicOrigin = Origin & {
 	scheme?: string;
 	database?: string;
 	user?: string;
+	access_client_id?: string;
+	access_client_secret?: string;
```

Comment:
This one section is a bit odd to me.  We exclude `password` here, presumably because it's not meant to be public?  Should we also give `access_client_secret` the same treatment?

Reply:
You're right. Moved it to the other type alongside the password.